src/authStrategies/RemoteAuth.js

```diff
@@ -0,0 +1,254 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper');
+const archiver = require('archiver');
+const fsPromise = require('fs').promises;
+const schedule = require('node-schedule');
```

Comment:
This dependency is also missing. What's the benefit of using this versus a plain `setInterval`?